anime_downloader/sites/helpers/request.py

```diff
@@ -9,6 +9,7 @@
 
 from anime_downloader import session
 from anime_downloader.const import get_random_header
+from anime_downloader.sites.helpers import selescrape
```

Comment:
you still havent made the `try`, `except ImportError` block

Reply:
No, it won't break support. Just like how you can't use twist.moe without node that's how you can't use kissanime, kisscartoon etc. I will make some errors for debugging that report that SeleScrape can not be used on the current config/OS. If I make a try except, what should it use? cfscrape? Isn't it not working? Well, I will make it do that just in case cfscrape actually works. Also, is there anything else you want changed/fixed? I really want SeleScrape to get accepted this weekend.

Reply:
My bad, you said it will break in the other comment. Not this one.